Shared/MainLayout.razor

```diff
@@ -16,6 +16,9 @@
                 <!--Home-->
                 <LowerNavItem Icon="home" Title="Home" Link="" />
 
+                <!--Response-->
+                <LowerNavItem Icon="notes" Title="Response" Link="response" />
+
```

Comment:
This is useful for testing but should be removed from this PR since the user won't use the lower nav to navigate to this page but rather will be directed here after submitting the form on the index/request page